tests/matchers_unittest.py

```diff
@@ -18,75 +18,132 @@ def test_NotThisInt(self):
+        with self.assertRaises(ValueError):
+            testslide.matchers.IntGreaterThan("derp")
+        with self.assertRaises(ValueError):
+            testslide.matchers.IntGreaterThan(42.42)
 
     def test_IntGreaterOrEquals(self):
         self.assertEqual(testslide.matchers.IntGreaterOrEquals(21), 42)
         self.assertEqual(testslide.matchers.IntGreaterOrEquals(21), 21)
         self.assertNotEqual(testslide.matchers.IntGreaterOrEquals(21), 20)
         self.assertNotEqual(testslide.matchers.IntGreaterOrEquals(42), "derp")
+        with self.assertRaises(ValueError):
+            testslide.matchers.IntGreaterOrEquals("derp")
+        with self.assertRaises(ValueError):
+            testslide.matchers.IntGreaterOrEquals(42.42)
 
     def test_IntLessThan(self):
         self.assertEqual(testslide.matchers.IntLessThan(21), 20)
         self.assertNotEqual(testslide.matchers.IntLessThan(21), 21)
         self.assertNotEqual(testslide.matchers.IntLessThan(21), 22)
         self.assertNotEqual(testslide.matchers.IntLessThan(42), "derp")
+        with self.assertRaises(ValueError):
+            testslide.matchers.IntLessThan("derp")
+        with self.assertRaises(ValueError):
+            testslide.matchers.IntLessThan(42.42)
 
     def test_IntLessOrEquals(self):
         self.assertEqual(testslide.matchers.IntLessOrEquals(21), 20)
         self.assertEqual(testslide.matchers.IntLessOrEquals(21), 21)
         self.assertNotEqual(testslide.matchers.IntLessOrEquals(21), 22)
         self.assertNotEqual(testslide.matchers.IntLessOrEquals(42), "derp")
+        with self.assertRaises(ValueError):
+            testslide.matchers.IntLessOrEquals("derp")
+        with self.assertRaises(ValueError):
+            testslide.matchers.IntLessOrEquals(42.42)
 
 
 class FloatMatcherTest(testslide.TestCase):
+    def testAnyFloat(self):
+        self.assertEqual(testslide.matchers.AnyFloat(), 666.0)
+        self.assertEqual(testslide.matchers.AnyFloat(), 4.2)
+        self.assertNotEqual(testslide.matchers.AnyFloat(), "derp")
+
     def test_NotThisFloat(self):
         self.assertEqual(testslide.matchers.NotThisFloat(66.6), 4.2)
         self.assertNotEqual(testslide.matchers.NotThisFloat(6.9), 6.9)
         self.assertNotEqual(testslide.matchers.NotThisFloat(4.2), "derp")
+        with self.assertRaises(ValueError):
+            testslide.matchers.NotThisFloat("derp")
+        with self.assertRaises(ValueError):
+            testslide.matchers.NotThisFloat(10)
 
     def test_FloatBetween(self):
-        self.assertEqual(testslide.matchers.FloatBetween(2.1, 666), 4.2)
-        self.assertEqual(testslide.matchers.FloatBetween(2.1, 666), 2.1)
-        self.assertEqual(testslide.matchers.FloatBetween(21, 66.6), 66.6)
+        self.assertEqual(testslide.matchers.FloatBetween(2.1, 666.0), 4.2)
+        self.assertEqual(testslide.matchers.FloatBetween(2.1, 666.0), 2.1)
+        self.assertEqual(testslide.matchers.FloatBetween(21.0, 66.6), 66.6)
```

Comment:
This is where the tests were updated as the argument used `666` was not of type `float`.

Reply:
I need to go through all the PR to fully review it.

a suggestion that I would provide, by checking this test, I'd that it would consider acceptable to have integers in the bounds of floatbeetween,after all the intention is to get numbers (floats) within 2 ranges

most probably the same comment will apply to greater and lesser matches as well